packages/web/src/components/nav/desktop/PlaylistLibrary.tsx

```diff
@@ -39,6 +43,54 @@ type PlaylistLibraryProps = {
   onClickNavLinkWithAccount: () => void
 }
 
+type LibraryContentsLevelProps = {
+  contents: PlaylistLibraryType['contents']
+  playlists: { [id: number]: AccountCollection }
+  renderPlaylist: (playlist: AccountCollection) => void
+  renderExplorePlaylist: (playlist: SmartCollection) => void
+  renderFolder: (folder: PlaylistLibraryFolder) => void
+}
+
+// Function component for rendering a single level of the playlist library. Playlist library consists of up to two content levels (root + inside a folder)
+const LibraryContentsLevel = ({
+  contents,
+  playlists,
+  renderPlaylist,
+  renderExplorePlaylist,
+  renderFolder
+}: LibraryContentsLevelProps) => {
+  console.log(contents, 'contents')
+  return (
+    <>
+      {contents.map(content => {
+        switch (content.type) {
+          case 'explore_playlist': {
+            const playlist = SMART_COLLECTION_MAP[content.playlist_id]
+            if (!playlist) return null
+            return renderExplorePlaylist(playlist)
+          }
+          case 'playlist': {
+            const playlist = playlists[content.playlist_id]
+            return renderPlaylist(playlist)
+          }
+          case 'temp_playlist': {
+            try {
+              const playlist = playlists[parseInt(content.playlist_id)]
+              return renderPlaylist(playlist)
+            } catch (e) {
+              console.debug(e)
```

Comment:
kept this since it was here before but not sure why?

Reply:
I don't think this actually could ever throw given how it's returned. parseInt just returns NaN and doesn't throw. I think save to remove.